docs/products/compute/kubernetes/get-started/index.md

```diff
@@ -47,47 +47,78 @@ Visit the [Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/inst
 
-1. When you are satisfied with the configuration of your cluster, click the **Create Cluster** button on the right hand side of the screen. Your cluster's detail page appears, and your Node Pools are listed on this page. From this page, you can [edit your existing Node Pools](#edit-or-remove-existing-node-pools), [access your Kubeconfig file](#access-and-download-your-kubeconfig), and view an overview of your cluster's resource details.
+1.  When you are satisfied with the configuration of your cluster, click the **Create Cluster** button on the right hand side of the screen. Your cluster's detail page appears, and your Node Pools are listed on this page. From this page, you can [edit your existing Node Pools](#edit-or-remove-existing-node-pools), [access your Kubeconfig file](#access-and-download-your-kubeconfig), and view an overview of your cluster's resource details.
 
 ### Access and Download your kubeconfig
 
-1. To access your cluster's kubeconfig, log in to your Cloud Manager account and navigate to the **Kubernetes** section.
+1.  To access your cluster's kubeconfig, log in to your Cloud Manager account and navigate to the **Kubernetes** section.
 
-1. From the Kubernetes listing page, click on your cluster's **more options ellipsis** and select **Download kubeconfig**. The file is saved to your computer's `Downloads` folder.
+1.  From the Kubernetes listing page, click on your cluster's **more options ellipsis** and select **Download kubeconfig**. The file is saved to your computer's `Downloads` folder.
 
-1. Open a terminal shell and save your kubeconfig file's path to the `$KUBECONFIG` environment variable. In the example command, the kubeconfig file is located in the `Downloads` folder, but you should alter this line with this folder's location on your computer:
+1.  Open a terminal shell and save your kubeconfig file's path to the `$KUBECONFIG` environment variable. In the example command, the kubeconfig file is located in the `Downloads` folder, but you should alter this line with this folder's location on your computer:
 
     ```command
     export KUBECONFIG=~/Downloads/kubeconfig.yaml
     ```
 
-1. View your cluster's nodes using kubectl.
+1.  View your cluster's nodes using kubectl.
 
     ```command
     kubectl get nodes
     ```
 
 ## General Network and Firewall Information
 
-{{% content "lke-network-firewall-information-shortguide" %}}
+In an LKE cluster, some entities and services are only accessible from within that cluster while others are publicly accessible (reachable from the internet).
+
+**Private (accessible only within the cluster)**
+
+- Pod IPs, which use a per-cluster virtual network in the range 10.2.0.0/16
+- ClusterIP Services, which use a per-cluster virtual network in the range 10.128.0.0/16
+
+**Public (accessible over the internet)**
+
+- NodePort Services, which listen on all Nodes with ports in the range 30000-32768.
```

Comment:
Bullet list under Private doesn't use periods, but under Public, it does. Consider changing one or other for consistency.  :)